packages/ui-library/src/components/options-menu/index.tsx

```diff
@@ -116,13 +118,13 @@ const OptionsMenuItems = ({ options, handleClose, anchorEl }: OptionsMenuItemsPr
                 {secondaryLabel && (
                   <Typography
                     variant="bodyExtraSmall"
-                    color={itemColor ? `${itemColor}.main` : colors[mode].typography.typo3}
+                    color={itemColor ? `${itemColor}.dark` : colors[mode].typography.typo3}
                   >
                     {secondaryLabel}
                   </Typography>
                 )}
               </ContainerBox>
-              {control && <Box sx={{ marginLeft: 'auto' }}>{control}</Box>}
```

Comment:
this doesnt break other stuff?

Reply:
I've tested all implementations of this component and works fine, I found this style redundant. The label container is already fullWidth, so this control should always be pushed to the right